cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-what-is-rosa.adoc

```diff
@@ -14,18 +14,174 @@ Red Hat OpenShift Service on AWS (ROSA) is a fully-managed turnkey application p
+
+== Support
+ROSA is supported by Red Hat and AWS, and you can contact the support team from either company to begin troubleshooting. Any escalations will be facilitated by Red Hat and AWS to engage the best team to address the issue.
+
+* link:https://access.redhat.com/[Red Hat Support]
+* link:https://aws.amazon.com/premiumsupport/[AWS Support]
+** AWS support customers must have a valid AWS support contract
+
+You can also visit the link:http://access.redhat.com/[Red Hat Customer Portal] to search or browse through the Red Hat knowledge base of articles and solutions relating to Red Hat products or submit a support case to Red Hat Support.
+
+You can also open a ticket directly from the link:https://console.redhat.com/openshift[Red Hat OpenShift Cluster Manager]. See the xref:../../support/getting-support.adoc#getting-support[ROSA support documentation] for more details about obtaining support.
+
+If a ROSA cluster is not upgraded before the "end of life" date, the cluster continues to operate in a limited support status. The SLA for that cluster will no longer be applicable, but you can still get support for that cluster. See the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[limited support status]documentation for more details.
+
+== Service-level agreement (SLA)
+Refer to the xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-sla_rosa-service-definition[ROSA SLA] page for details.
+
+== Notifications and communication
+Red Hat will provide notifications regarding new features, updates, and scheduled maintenance through email, including AWS updates and email,  and the Red Hat console service log.
+
+== Open Service Broker for AWS (OBSA)  
+You can use OSBA with ROSA.  See link:https://aws.amazon.com/partners/servicebroker/[Open Service Broker for AWS] for more information.
+
+The preferred method is the more recent link:https://github.com/aws-controllers-k8s/community[AWS Controller for Kubernetes].
+
+== Offboarding  
+Customers can stop using ROSA at any time and move their applications to on-premise, a private cloud, or other cloud providers. Standard reserved instances (RI) policy applies for unused RI.   
+
+== Authentication
+ROSA supports the following authentication mechanisms: OpenID Connect (a profile of OAuth2), Google OAuth, GitHub OAuth, GitLab, and LDAP.
+
+== SRE cluster access
+All SRE cluster access is secured by MFA. See xref:../../rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-sre-access_rosa-policy-process-security[SRE access] for more details.
```

Comment:
Link probably needs to now go here: https://70661--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-sre-access